bridge/client/app/_components/ktb-project-settings/ktb-project-settings.component.ts

```diff
@@ -209,26 +209,31 @@ export class KtbProjectSettingsComponent implements OnInit, OnDestroy, PendingCh
 
     this.dataService
       .createProjectExtended(projectName, shipyardBase64, this.gitDataExtended ?? undefined)
-      .pipe(switchMap(() => this.dataService.loadProjects()))
-      .subscribe(
-        () => {
-          this.projectName = projectName;
-          this.isCreatingProjectInProgress = false;
-          this.isProjectFormTouched = false;
-
-          this.router.navigate(['/', 'project', this.projectName, 'settings', 'project'], {
-            queryParams: { created: true },
-          });
-        },
-        (err) => {
+      .pipe(
+        map(() => true),
+        catchError((err) => {
           const errorMessage = err.error || 'please, check the logs of resource-service';
           this.notificationsService.addNotification(
             NotificationType.ERROR,
             `The project could not be created: ${errorMessage}.`
           );
           this.isCreatingProjectInProgress = false;
+          return of(false);
+        })
+      )
+      .subscribe((success) => {
+        if (!success) {
+          return;
         }
-      );
+        this.dataService.loadProjects();
```

Comment:
Because we don't wait for the request anymore before navigating, the project-not-found message shows up for some milliseconds before the project settings page is shown, after creating a project.

Reply:
Where is the project-not-found message created?